### ISSUE TITLE
Index out of range for second Electrical Series

### ISSUE THREAD

**Issue by sportnoah14** [NeurodataWithoutBorders/nwb-jupyter-widgets#130]
Hi, I created two ElectricalSeries objects in my nwb file. Each of them has their own electrode table region (one goes from 0-166, another goes from 167-183). Both acquisitions show up in the widget, but the acquisition ranging from 167-183 produces the following error: `ValueError: Index (167) out of range (0-15)`. I don't think this is an error with the table regions but I could be mistaken. I also checked that both acquisitions have the correct number of channels and the file worked with `pynwb.validate()`. Any help would be useful. Thank you

Noah

**Comment by bendichter** [comment]
Hi Noah, thanks for reaching out. Would you be able to share the file?

**Comment by sportnoah14** [comment]
Hi Ben,

Yes I can share the file. I have the file on a server and can send you a link to it. Is there an e-mail address I can send the link for the file to you?

**Comment by bendichter** [comment]
bdichter@lbl.gov

**Comment by bendichter** [comment]
@sportnoah14 yup, that was a bug. Thanks. The it's fixed in the latest. Install with:
```
$ pip uninstall nwbwidgets & pip install git+https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets.git
```

**Comment by bendichter** [comment]
actually... that's going to break some other things. I will fix it but I need to think about how best to do it

**Comment by bendichter** [comment]
fixed with #132
